content/en/user-guide/aws/cloudformation/index.md

```diff
@@ -160,178 +160,232 @@ When utilizing the Community Edition, any resources within the stack that are no
-| AWS::EC2::SecurityGroup                     |     ✅ |     ✅ |      - |
-| AWS::EC2::Subnet                            |     ✅ |     ✅ |      - |
-| AWS::EC2::SubnetRouteTableAssociation       |     ✅ |     ✅ |      - |
-| AWS::EC2::VPC                               |     ✅ |     ✅ |      - |
-| AWS::EC2::VPCGatewayAttachment              |     ✅ |      - |      - |
-| AWS::ECR::Repository                        |     ✅ |     ✅ |      - |
-| AWS::Elasticsearch::Domain                  |     ✅ |     ✅ |      - |
-| AWS::Events::Connection                     |     ✅ |     ✅ |      - |
-| AWS::Events::EventBus                       |     ✅ |     ✅ |      - |
-| AWS::Events::EventBusPolicy                 |     ✅ |     ✅ |      - |
-| AWS::Events::Rule                           |     ✅ |     ✅ |      - |
-| AWS::IAM::Group                             |     ✅ |     ✅ |      - |
-| AWS::IAM::InstanceProfile                   |     ✅ |     ✅ |      - |
-| AWS::IAM::ManagedPolicy                     |     ✅ |      - |      - |
-| AWS::IAM::Policy                            |     ✅ |     ✅ |      - |
-| AWS::IAM::Role                              |     ✅ |     ✅ |     ✅ |
-| AWS::IAM::ServiceLinkedRole                 |     ✅ |     ✅ |      - |
-| AWS::IAM::User                              |     ✅ |     ✅ |      - |
-| AWS::KMS::Alias                             |     ✅ |     ✅ |      - |
-| AWS::KMS::Key                               |     ✅ |     ✅ |      - |
-| AWS::Kinesis::Stream                        |     ✅ |     ✅ |      - |
-| AWS::Kinesis::StreamConsumer                |     ✅ |     ✅ |      - |
-| AWS::KinesisFirehose::DeliveryStream        |     ✅ |     ✅ |      - |
-| AWS::Lambda::EventInvokeConfig              |     ✅ |     ✅ |      - |
-| AWS::Lambda::EventSourceMapping             |     ✅ |      - |      - |
-| AWS::Lambda::Function                       |     ✅ |     ✅ |     ✅ |
-| AWS::Lambda::Permission                     |     ✅ |      - |      - |
-| AWS::Lambda::Version                        |     ✅ |      - |      - |
-| AWS::Logs::LogGroup                         |     ✅ |     ✅ |      - |
-| AWS::Logs::LogStream                        |     ✅ |     ✅ |      - |
-| AWS::Logs::SubscriptionFilter               |     ✅ |     ✅ |      - |
-| AWS::OpenSearchService::Domain              |     ✅ |     ✅ |      - |
-| AWS::Redshift::Cluster                      |     ✅ |      - |      - |
-| AWS::ResourceGroups::Group                  |     ✅ |     ✅ |      - |
-| AWS::Route53::RecordSet                     |     ✅ |      - |      - |
-| AWS::S3::Bucket                             |     ✅ |     ✅ |      - |
-| AWS::S3::BucketPolicy                       |     ✅ |     ✅ |      - |
-| AWS::SNS::Subscription                      |     ✅ |     ✅ |      - |
-| AWS::SNS::Topic                             |     ✅ |     ✅ |      - |
-| AWS::SNS::TopicPolicy                       |     ✅ |     ✅ |      - |
```

Comment:
subscription and topicpolicy  are currently missing 

cc @Morijarti

Reply:
I'll start with their implementation. I'm not sure how I missed them.

Reply:
https://github.com/localstack/localstack/pull/10359